WeakAuras/GenericTrigger.lua

```diff
@@ -2964,6 +2965,10 @@ do
           WeakAuras.ScanEvents("BigWigs_StopBar", id)
         end
       end
+    elseif event == "BigWigs_SetStage" then
+      local addon, stage = ...
+      currentStage = stage
+      WeakAuras.ScanEvents("BigWigs_SetStage", ...)
```

Comment:
Is this event also send on encounter end/reset? (I would assume not, and then it probably requires adjusting the code in the BigWigs_OnBossDisable above?

Reply:
Yea, I discussed this with Funkeh and we probablly will reset the stage number on boss resets/wins. We don't do this rightnow.

Reply:
to add: we do always set it to 1 on loading a boss module/engaging right now.